Packages.props

```diff
@@ -64,11 +64,11 @@
 	<!-- When updating version of Dependencies in the below section, please also update the version in the following files:
 		packages\Microsoft.SqlTools.ManagedBatchParser\Microsoft.SqlTools.ManagedBatchParser.nuspec-->
 	<ItemGroup>
-		<PackageReference Update="Azure.Identity" Version="1.10.2" />
-		<PackageReference Update="Microsoft.Data.SqlClient" Version="5.1.3" />
+		<PackageReference Update="Azure.Identity" Version="1.10.3" />
+		<PackageReference Update="Microsoft.Data.SqlClient" Version="5.1.4" />
 		<PackageReference Update="Microsoft.SqlServer.SqlManagementObjects" Version="170.18.0" />
 		<PackageReference Update="Newtonsoft.Json" Version="13.0.3" />
-		<PackageReference Update="System.Configuration.ConfigurationManager" Version="7.0.0" />
+		<PackageReference Update="System.Configuration.ConfigurationManager" Version="8.0.0" />
```

Comment:
Was it a required change to resolve some conflict? If yes, which package references 8.0.0?

Reply:
updating DacFx version to latest preview raised errors on build for the version downgrade of above three packages. I know we upgraded AzureIdentity and MDS in dacfx. Checking which package caused configurationManager version reference.

Reply:
Actually all are for DacFx only:

error NU1605:  MicrosoftSqlToolsServiceLayer -> Microsoft.SqlServer.DacFx 162.2.82-preview -> System.Runtime.Caching 8.0.0 -> Sys
tem.Configuration.ConfigurationManager (>= 8.0.0)

Reply:
Ok, we should've 'ideally' been on 7.0.0 till we used .NET 7.0, but I hope they're backwards compatible and nothing much has changed. For future ref, do not update this package in DacFx to be beyond .NET major version as sometimes they may bring in breaking changes.

We will soon plan on updating .NET to 8.0 to avoid any issues with this package.

Reply:
Got it, the reason for updating it is now DacFx supports .NET8.0 too.

Reply:
...or at the very least, you should always be bumping the major version whenever a dependency with a new major version is brought in so it adheres to semver rules. Otherwise you could end up breaking customers if they have a dependency on a specific major version of a package.

Reply:
those packages still have target for .net 6. updating it to 8.0.0 should not break anything

Reply:
if ADS is still on .net6/7 we can make the version of those packages for target .net6 6.0.0. for .net8 we can still use 8.0.0. I think that would make more sense

Reply:
STS is still net7, yeah. And that idea sounds great! Hopefully it isn't too much of a pain.